vsintegration/src/FSharp.Editor/Classification/ClassificationService.fs

```diff
@@ -117,23 +114,20 @@ type internal FSharpClassificationService
             | true, items -> addSemanticClassification sourceText targetSpan items outputResult
             | _ -> ()
 
-    static let toSemanticClassificationLookup (data: SemanticClassificationData) =
+    static let toSemanticClassificationLookup (d: SemanticClassificationData) =
         let lookup = System.Collections.Generic.Dictionary<int, ResizeArray<SemanticClassificationItem>>()
-        match data with
-        | None -> ()
-        | Some d ->
-            let f (dataItem: SemanticClassificationItem) =
-                let items =
-                    match lookup.TryGetValue dataItem.Range.StartLine with
-                    | true, items -> items
-                    | _ ->
-                        let items = ResizeArray()
-                        lookup.[dataItem.Range.StartLine] <- items
-                        items
-
-                items.Add dataItem
-
-            d.ForEach(f)
+        let f (dataItem: SemanticClassificationItem) =
+            let items =
+                match lookup.TryGetValue dataItem.Range.StartLine with
+                | true, items -> items
+                | _ ->
+                    let items = ResizeArray()
+                    lookup.[dataItem.Range.StartLine] <- items
+                    items
+
+            items.Add dataItem
+
+        d.ForEach(f)
```

Comment:
Stylistically I'd prefer `d.ForEach(fun dataItem -> ...)` rather than using a local function.  But that's unrelated to this PR